src/ResourceManager/Sql/Commands.Sql/Auditing/Cmdlet/SetAzureSqlDatabaseAuditingPolicy.cs

```diff
@@ -135,6 +144,10 @@ private void ApplyUserInputToBlobAuditingModel(DatabaseBlobAuditingPolicyModel m
                 model.AuditAction = AuditAction;
             }
 
+            if (EventType != null) // Event types are relevant only for Table auditing
```

Comment:
This will be a breaking change. I will finish handling the parameters in another task as agreed with Adam and Mark.